build.sbt

```diff
@@ -26,6 +27,41 @@ def common: Seq[Setting[_]] =
     Compile / javacOptions ++= Seq("-encoding", "UTF-8"),
     Compile / scalacOptions ++= Seq("-encoding", "UTF-8", "-deprecation"))
 
+lazy val devMode = project
+  .in(file("devmode"))
```

Comment:
New module to provide integration with testcontainers for dev-mode. 

Code here is a copy of KalixProxyContainer that we have for integration tests with some small differences.